### PR TITLE
add froschmc

### PR DESCRIPTION
**Pull request by FakeBlack1**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Review by DoJapHD — CHANGES_REQUESTED** [review]
Still the same changes as requested in the last PR

**Comment by FakeBlack1** [comment]
hat es funktioniert?

File: minecraft_servers/froschmc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "froschmc",
+  "nice_name": "FROSCHMC",
+  "direct_ip": "froschmc.de",
+  "server_wildcards": [
+    "%.froschmc.de"
```
Comment (DoJapHD) [inline minecraft_servers/froschmc/manifest.json:6]: Not a real wildcard

File: minecraft_servers/froschmc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://www.froschmc.de",
```
Comment (DoJapHD) [inline minecraft_servers/froschmc/manifest.json:12]: Not reachable

File: minecraft_servers/froschmc/manifest.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "web_shop": "https://froschmc.de",
```
Comment (DoJapHD) [inline minecraft_servers/froschmc/manifest.json:13]: Not reachable

File: minecraft_servers/froschmc/manifest.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "discord": "https://discord.gg/KKHWZpRZuM",
+    "teamspeak": ""
```
Comment (DoJapHD) [inline minecraft_servers/froschmc/manifest.json:15]: Remove

File: minecraft_servers/froschmc/manifest.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    "primary": "#0095ed",
+    "background": "#00182e",
+    "text": "#ffffff"
+  },
+  "location": {
+    "city": "Frankfurt",
+    "country": "Germany",
+    "country_code": "DE"
+  },
+  "yt_trailer": ""
```
Comment (DoJapHD) [inline minecraft_servers/froschmc/manifest.json:35]: Remove

File: minecraft_servers/froschmc/manifest.json [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  ],
+  "social": {
+    "web": "https://www.froschmc.de",
+    "web_shop": "https://froschmc.de",
+    "discord": "https://discord.gg/KKHWZpRZuM",
+    "teamspeak": ""
+  },
+  "chat": {
+    "message_formats": [
+      "^(|§8»\\n)(§[a-f0-9]§?l?(?<rank>\\S+))§r §8\\| §7(?<sender>[a-zA-Z0-9_]{2,16})§r §8» §7(?<message>.*)(|\\n§8»)$"
```
Comment (DoJapHD) [inline minecraft_servers/froschmc/manifest.json:19]: Are you sure that's your regex?

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/froschmc/background.png
Comment (DoJapHD) [inline minecraft_servers/froschmc/background.png:1]: Compress image

File: minecraft_servers/froschmc/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/froschmc/background@2x.png:1]: Compress image

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Pull request closed without merging**